src/main/java/net/dv8tion/jda/api/EmbedBuilder.java

```diff
@@ -386,6 +409,17 @@ public EmbedBuilder setTimestamp(@Nullable TemporalAccessor temporal)
         return this;
     }
 
+    /**
+     * The {@link java.awt.Color Color} of the embed
+     *
+     * @return Color the builder has been set with.
+     */
+    @Nonnull
+    public java.awt.Color getColor()
```

Comment:
You should also add `getColorRaw`, keep it in line with the rest of JDA

Reply:
thanks

Reply:
We already import java.awt.Color so we don't need to use FQDN

Reply:
All fixed, thanks